tests/test_gbdesAstrometricFit.py

```diff
@@ -340,6 +340,7 @@ def _make_sourceCat(cls, starIds, starRas, starDecs, trueWCSs, inScienceFraction
                     "hsmPsfMoments_flag",
                     "apFlux_12_0_flag",
                     "extendedness",
+                    "sizeExtendedness",
```

Comment:
Any reason you have both of these here?

Reply:
We still want to propagate the what I call "flux extendedness" values, but switch from using one to another. At some future point, I'd like to make `slot_extendedness` that can point to either, but not in the near future.